site/en/tutorials/generative/dcgan.ipynb

```diff
@@ -273,32 +246,32 @@
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "colab": {},
-        "colab_type": "code",
         "id": "6bpTcDqoLWjY"
       },
       "outputs": [],
       "source": [
+        "# Random Normal model initializers as per Dc-gan paper\n",
+        "RN_initializers=tf.keras.initializers.RandomNormal(mean=0.0,stddev=0.02)\n\n",
```

Comment:
That's a great observation about the zero-centered Normal, thanks for bringing this up.

I have a few style-related suggestions, if you don't mind: 

- Change the DCGAN spelling from "Dc-gan" (source: https://arxiv.org/pdf/1511.06434.pdf) (in the code comment)
- Change to `weight initializer`, instead of `model initializers` (also in the code comment)
- Add a blank line before the generator method
- Add a space in the params section in `RandomNormal()` after a comma (I think it's PEP 8: https://www.python.org/dev/peps/pep-0008/)

```suggestion
        "# Random Normal weight initializer as per the DCGAN paper\n",
        "RN_initializers=tf.keras.initializers.RandomNormal(mean=0.0, stddev=0.02)\n\n",
        
```

Also, since it's a weight initializer (singular), maybe you should rename the variable to `RN_initializer` without the `s`.

Anyway, these are just recommendations 👍